packages/react-native/Libraries/Text/RCTTextAttributes.mm

```diff
@@ -8,16 +8,85 @@
 #import <React/RCTTextAttributes.h>
 
 #import <React/RCTAssert.h>
+#import <React/RCTCursor.h> // [macOS]
 #import <React/RCTFont.h>
 #import <React/RCTLog.h>
 
-#if TARGET_OS_OSX // [macOS
-#import <React/RCTCursor.h>
-#endif // macOS]
-
 NSString *const RCTTextAttributesIsHighlightedAttributeName = @"RCTTextAttributesIsHighlightedAttributeName";
 NSString *const RCTTextAttributesTagAttributeName = @"RCTTextAttributesTagAttributeName";
 
+#if TARGET_OS_OSX // [macOS
+// Duplicate this function here as RCTTextAttributes doesn't depend on RCTView
```

Comment:
Can we put this function in a common module instead of duplicating?

Reply:
It's easy to share a header like RCTCursor. It was not so easy to share a method implementation. I didn't want to make more pod spec changes/diffs in RNM, so I thought duplicating was the lesser of the two evils.

Reply:
Ended up making the method inline, which also worked.